.circleci/verdaccio.yml

```diff
@@ -0,0 +1,44 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
+  npmjs:
+    url: https://registry.yarnpkg.com/
```

Comment:
Why is this not `https://registry.npmjs.org/`? Plus having the variable as `npmjs` while the hostname is yarnpkg is confusing

Reply:
This came about from [this](https://github.com/facebook/react-native/pull/34577#discussion_r965885227) conversation. I don't mind either way, let me revert it back to `https://registry.npmjs.org/` 👍